qa/suites/upgrade/octopus-p2p/octopus-p2p-parallel/supported-all-distro/centos_latest.yaml

```diff
@@ -0,0 +1,2 @@
+os_type: centos
```

Comment:
supported-all-distro should be a symlink to qa/distros/supported-all-distros instead of a copy - it's out of date

Reply:
removed all but latest ubuntu